Footer block

sneharora1 opened the pull request:
Footer Block

Fix #9

Test URLs:
- Before: https://main--infosys--aemsites.hlx.page/iki
- After: https://footer--infosys--aemsites.hlx.page/iki

shrotia commented:
Overall looks good, just few trivial points:
1. I have set the font-size=10px through [PR ](https://github.com/aemsites/infosys/pull/16)as @anuj-adobe just pointed out that they IKI.html has followed the same, and we should also stick to it. Impact: please check all rem that we defined in this PR has any impact or need adjustment.
2. Remove the country list from the Document.
3. We may load /drafts/sneh/countrydropdown.json when user click on that Country dropdown
4. Also we can use absolute link i.e. https://footer--infosys--aemsites.hlx.page/drafts/sneh/countrydropdown.json in the word document [ as recommended here](https://www.aem.live/docs/faq#q55)

shrotia commented on scripts/blocks-utils.js:
```diff
@@ -0,0 +1,28 @@
+function createElement(tagname, className) {
```
To avoid confusion with the built-in document.createElement, you might consider renaming it to something more specific, like createStyledElement or createCustomElement. This can help prevent unintentional overrides or confusion when working with the standard DOM manipulation methods.

sneharora1 replied:
Changed the name to createCustomElement

sneharora1 commented:
> Overall looks good, just few trivial points:
> 
> 1. I have set the font-size=10px through [PR ](https://github.com/aemsites/infosys/pull/16)as @anuj-adobe just pointed out that IKI.html has followed the same, and we should also stick to it. Impact: please check all rem that we defined in this PR has any impact or need adjustment.
> 2. Please remove the country list from the Document.
> 3. We may load /drafts/sneh/countrydropdown.json when user click on that Country dropdown
> 4. Also we can use absolute link i.e. https://footer--infosys--aemsites.hlx.page/drafts/sneh/countrydropdown.json in the word document [ as recommended here](https://www.aem.live/docs/faq#q55)

addressed all comments
